pages/bank.js

```diff
@@ -35,17 +37,23 @@ export default function Bank() {
           description="Hack Club Bank provides a 501(c)(3) status-backed fund optimized for high school hackathons including invoicing, debit cards, check sending, pre-written legal forms, automated tax filing, and transparent finances. Get fiscal sponsorship designed to help you run a great organization."
           image="https://cloud-og86rfngo-hack-club-bot.vercel.app/0og_image-2.png"
         />
-        <style>{ styles }</style>
+        <style>{styles}</style>
         <Box>
-          <Landing />
-          <Features />
+          {isGivebutter ? <Landing /> : <Landing showButton />}
+          {isGivebutter ? <Features partner={true} /> : <Features partner={false} />}
           <Testimonials />
           <Nonprofits />
-          <Everything />
-          <Start />
+          {isGivebutter ? (
+            <Everything fee="10" partner={true} />
+          ) : (
+            <Everything fee="7" partner={false} />
+          )}
+          {!isGivebutter && <Start />}
         </Box>
       </Box>
       <Footer dark key="footer" />
     </>
   )
 }
+
+export default Bank
```

Comment:
Any reason for this change? It's cool with me but just curious

Reply:
No reason, I was just experimenting but forgot to change it back.